op-e2e/Makefile

```diff
@@ -1,14 +1,29 @@
+test: pre-test
+	go test -v ./...
+
+cannon-prestate:
+	make -C .. cannon-prestate
+
 devnet-allocs:
-	PYTHONPATH=../bedrock-devnet python3 ../bedrock-devnet/main.py --monorepo-dir=.. --allocs
+	make -C .. devnet-allocs
```

Comment:
I think this actually depends on cannon-prestate existing since the prestate is required when deploying the fault dispute contract so probably worth stating that in the makefile.

Reply:
oh it does, I should probably leave a comment, I had to switch the order of `pre-test` so it depends on `cannon` before `allocs`.